### PR TITLE
Fixes some possible NPEs in PotionEventHandler

### PR DESCRIPTION
**Pull request by keraldi**
#1617

### CODE REVIEW

File: src/main/java/WayofTime/bloodmagic/potion/PotionEventHandlers.java [hunk 1]
```diff
@@ -93,19 +93,23 @@ public static void onEntityUpdate(LivingEvent.LivingUpdateEvent event) {
...
 //        }
         List<EntityLivingBase> noGravityList = noGravityListMap.getOrDefault(event.getEntityLiving().getEntityWorld(), Lists.newArrayList());
-        if (eventEntityLiving.isPotionActive(RegistrarBloodMagic.SUSPENDED) && !eventEntityLiving.hasNoGravity()) {
-            eventEntityLiving.setNoGravity(true);
-            noGravityList.add(eventEntityLiving);
-        } else if (noGravityList.contains(eventEntityLiving)) {
-            eventEntityLiving.setNoGravity(false);
-            noGravityList.remove(eventEntityLiving);
-        }
+        if (noGravityList != null)
```
Comment (JonesiBlitz) [inline src/main/java/WayofTime/bloodmagic/potion/PotionEventHandlers.java]: No opening bracket for this case.

Comment (keraldi) [reply src/main/java/WayofTime/bloodmagic/potion/PotionEventHandlers.java]: No opening bracket needed, as there is only one statement below it (another conditional).

Comment (Keraldi) [reply src/main/java/WayofTime/bloodmagic/potion/PotionEventHandlers.java]: No opening bracket needed, as there is only one statement below it (another conditional).

Comment (TehNut) [reply src/main/java/WayofTime/bloodmagic/potion/PotionEventHandlers.java]: A multi-line statement should still be wrapped for readability

File: src/main/java/WayofTime/bloodmagic/potion/PotionEventHandlers.java [hunk 2]
```diff
@@ -93,19 +93,24 @@ public static void onEntityUpdate(LivingEvent.LivingUpdateEvent event) {
...
+                noGravityList.add(eventEntityLiving);
+            } else if (noGravityList.contains(eventEntityLiving)) {
+                eventEntityLiving.setNoGravity(false);
+                noGravityList.remove(eventEntityLiving);
+            }
+        }
+        
+        if (eventEntityLiving.isPotionActive(RegistrarBloodMagic.GROUNDED)) {
+            PotionEffect activeEffect = eventEntityLiving.getActivePotionEffect(RegistrarBloodMagic.GROUNDED);
+            if (activeEffect != null)
```
Comment (JonesiBlitz) [inline src/main/java/WayofTime/bloodmagic/potion/PotionEventHandlers.java]: Another multi line block is needed here.
While single line if statements don’t need wrapped it is almost easier to just have all if statements Wrapped for clarity. It is a team level decision though at most places.

**Pull request merged**